### DISCUSSION TITLE
Prøv å installere R og RStudio på datamaskinen før kurset

### DISCUSSION THREAD

**Discussion by hbuis** [hbuis/R_kurs_demo#2] [General]
Hei,

Før du kommer på kurs så er det flott hvis du kan installere R på datamaskinen.

Du kan laste ned Rstudio her: https://posit.co/download/rstudio-desktop/ 

eller du kan få det via programvaresenteret på din institusjon. 

Se også PDF filen under for instruksjoner rundt oppsettet av RStudio og installasjon av pakker (viktig!). 

[R og RStudio.pdf](https://github.com/hbuis/R_kurs_demo/files/10573318/R.og.RStudio.pdf)

**Comment by OisteinA** [comment]
Installasjon blir litt annerledes for de som bruker Mac. Bør vi lage en egen bruksanvisning for dem? Ellers brukte jeg denne YouTube-videoen da jeg installerte på min Mac https://www.youtube.com/watch?v=bLSnXURs1YA

**Reply by hbuis** [reply]
Denne videoen virker ganske grei og viser trinn for trinn hvordan vi installerer R på Mac. Vet ikke om vi trenger å lage en egen bruksanvisning...

Foreslår at vi inviterer deltakere ca. 2 uker før kurset begynner og ber dem om å installere R og RStudio. Så kan de gi en tilbakemelding hvis de vil ha en bruksanvisning.

**Reply by 7Nadia7** [reply]
Da prøver jeg å laste ned ved å bruke denne linken. Jeg har Mac. Takk for videolinken!

**Comment by anitalopezpedersen** [comment]
JEg sliter med å laste ned Rstudio på jobbmaskinen min - flere som har samme problem? Jeg blir stoppet av admin. på maskinen.

**Reply by hbuis** [reply]
Hei Anita!

På UiO sine sider så leser jeg at lokal IT i fakultet/avdelingen kan bistå med dette. Er du kjent med de? Kanskje hjelper denne nettsiden med det --> https://weblogin.uio.no/simplesaml/module.php/delegate/login.php?AuthState=_9304eb69e0e2cd4c6287d1d5a7cc27f59b717bbad1%3Ahttps%3A%2F%2Fweblogin.uio.no%2Fsimplesaml%2Fsaml2%2Fidp%2FSSOService.php%3Fspentityid%3Dhttps%253A%252F%252Fwww-int.uio.no%26cookieTime%3D1677682654%26RelayState%3Dhttps%253A%252F%252Fwww-int.uio.no%252Ftjenester%252Fit%252Fmaskin%252Fprogramvare%252Fhjelp%252Fprogramvaredrift%252F%253FauthTicket%253DHSlYTa-m8GELU8pig-ufng

Prøv også IT-hjelp på UiO --> https://www.uio.no/tjenester/it/kontakt/helpdesk/ 

Jeg også leser at Student-IT kan hjelpe med å installere R --> https://www.mn.uio.no/tjenester/it/hjelp/student-it/ 

Dersom disse ikke kan hjelpe deg så kan du også nå R via UiO Programkiosk --> https://www.uio.no/tjenester/it/maskin/programvare/programkiosk/ 

Mvh
Haris

**Reply by anitalopezpedersen** [reply]
Takk for raskt svar - jeg tror jeg på laste det ned via softwarecenter, tror det gikk, men jeg får beskjed om at jeg har RSTudio 2022.07.2+576 og ikke 2022.12.0+353, har det noe å si?

**Reply by hbuis** [reply]
Jeg tror ikke det skal ha noe å si. Ser her at den versjonen av R ble utgitt den 19/9-2022. Kan ikke se at den nyere versjonen er særlig annerledes. Så jeg tror at det skal gå helt fint.

![image](https://user-images.githubusercontent.com/124052481/222184571-a7ff86fc-8277-4acf-b29d-a4043481bbac.png)

**Reply by anitalopezpedersen** [reply]
Supert! skal vi laste ned noen pakker  - lavaan?

**Reply by hbuis** [reply]
Du kan laste ned pakker hvis du vil, men dette er en del av kurset. Vi går gjennom pakker på første dag og ellers laster vi ned pakker etter behov når vi går videre i kurset. Jeg viser i kurset hvilke pakker dere trenger og hvilke funksjoner som er aktuelle å bruke med konkrete eksempler.

Du kan laste ned lavaan og prøve ut pakken hvis du vil :)

**Reply by anitalopezpedersen** [reply]
Jepp, lastet ned lavaan. men ikke sikker på om jeg har gjort det riktig. men kan prøve ut litt. Takk for raske svar, Haris!

**Reply by hbuis** [reply]
Det kan vi ta en kikk på i kurset!

Bare hyggelig :)

**Comment by DieuwertenBraak** [comment]
Hei!

Jeg har R/Rstudio fra før av på maskinen - men R 4.2.1. Er det best jeg installerer alt på nytt eller går det bra?

Dieuwer

**Reply by hbuis** [reply]
Hei Dieuwer,

Dette går helt fint! Du trenger ikke å installere den nye versjonen.

Mvh
Haris

**Reply by DieuwertenBraak** [reply]
T.O. "package ‘narniar’ is not available for this version of R"

**Reply by hbuis** [reply]
Hi Dieuwer!

Did you try to install Naniar? You need a version newer than 3.1.2 to use this package. 

![image](https://user-images.githubusercontent.com/124052481/223449320-9a052d84-2930-4ea6-9a89-7ec9c7c9baed.png)

**Comment by DieuwertenBraak** [comment]
See - I need new glasses "narniar" -> "naniar". Now it works - thanks!

**Reply by hbuis** [reply]
👍
